controllers/cloud.redhat.com/providers/kafka/strimzi.go

```diff
@@ -296,18 +335,88 @@ func (s *strimziProvider) Provide(app *crd.ClowdApp, c *config.AppConfig) error
 		}
 	}
 
+	ku := &strimzi.KafkaUser{}
```

Comment:
For readability, can we move this logic into a private func named `createKafkaUser` that returns a `*strimzi.KafkaUserSpec`?

Reply:
I see no reason why not

Reply:
Actually this doesn't need to return the ku at all